src/presentation/pages/Settings.vue

```diff
@@ -54,12 +61,29 @@ async function userLogout() {
     router.push({ path: '/' });
   });
 }
+
+/**
+ * Deletes tool from the user
+ *
+ * @param toolId - id of the tool
+ */
+async function uninstallClicked(toolId: string) {
+  if (window.confirm(t('confirmations.uninstallEditorTool'))) {
+    await removeTool(toolId);
+  }
+}
 </script>
 
-<style scoped lang="postcss">
+<style scoped lang="postcss" module>
 @import '@/presentation/styles/typography.pcss';
 
 .marketplace {
   margin-top: var(--spacing-l);
 }
+
+.user__tool {
```

Comment:
if you're using CSS Modules, there is no reason to use BEM. You can use simple classes like `.tool`. CSS Modules will prevent conflict adding hashes to the class under the hood

Reply:
done